lizrd/scripts/grid.py

```diff
@@ -37,12 +37,21 @@
     except IndexError:
         raise ValueError("No config path specified. Aborting...")
 
+    try:
+        code_mirror_git_branch = sys.argv[2]
+    except IndexError:
+        code_mirror_git_branch = ""
+        assert (
+            CLUSTER_NAME == MachineBackend.LOCAL
+        ), f"No git branch with mirrored code specified. If you are calling grid.py directly on a cluster (not recommneded), supply any string as the second argument. Aborting..."
```

Comment:
'recommended' spelling mistake

Reply:
Also, I don't like passing git_branch_name as a param here. How about passing it in yaml?

Reply:
pogadamy